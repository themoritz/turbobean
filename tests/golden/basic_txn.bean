2020-01-01 open Assets:Cash
2020-01-01 open Expenses:Home

2020-01-02 * "Buy"
  Assets:Cash  -10 EUR
  Expenses:Home

