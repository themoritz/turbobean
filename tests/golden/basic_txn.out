Assets     -10 EUR
  Cash     -10 EUR
Expenses   10 EUR
  Home     10 EUR
